test/rest_api/test_storage_service.py

```diff
@@ -290,7 +298,7 @@ def verify_snapshot_details(expected):
                 verify_snapshot_details({
                     'key': tag1,
                     'value': [
-                        {'ks': ks0, 'cf': cf00, 'total': 1, 'live': 0},
+                        {'ks': ks0, 'cf': cf00, 'total': 0, 'live': 0},
```

Comment:
I think I don't understand what this "total" means... Why is it now 0?

Reply:
`total` sums the (disk allocated) size of the sstables in the snapshot.
If the table contained no data, `total` is zero (i.e. it doesn't not include the size of the `schema.cql` and `manifest.json` files that are in the snapshot directory.

`live` sums the size of sstables that are present both in the snapshot _and_ in the table base directory (they are hard-linked together).  It can be used to estimate how much space will be freed when deleting the snapshot.

BTW, it is hard to tell by the this api alone since `total - live` is not really the unique capacity in the snapshot since the sstables that aren't considered "live" may still be hardlinked in other snapshot, so deleting a particular snapshot will not necessarily free-up all the non-live space.  In theory, we could extend the api with a `unique` counter to the snapshot details that will hold the size of sstables that have a link-count of 1 and hence are unique to this particular snapshot.